microsoftgraph/microsoft-graph-explorer-v4
# Fix: interaction in progress

## Overview
This issue mainly affects users who have very slow internet.

Closes #956 

### Demo

N/A

### Notes

Shows a "Signing you out.." message as sign out occurs

## Testing Instructions

* Log in to GE
* Slow down your network by changing the network to slow 3G in the network tab of your chrome dev tools
* Sign out
* Notice the message and the absence of the button
* You will immediately be redirected to another page to finish the signing out. Once back your sign out will be complete
